esinstall/src/index.ts

```diff
@@ -302,7 +303,8 @@ export async function install(
         const aliasEntry = findMatchingAliasEntry(installAlias, specifier);
         return aliasEntry && aliasEntry.type === 'package' ? aliasEntry.to : specifier;
       })
-      .sort(),
+      .map((specifier) => specifier.replace(/(\/|\\)+$/, '')) // remove trailing slash from end of specifier (easier for Node to resolve)
+      .sort((a, b) => a.localeCompare(b, undefined, {numeric: true})),
```

Comment:
curious, why is this needed?

Reply:
I think I default to `localeCompare()` more out of habit these days (it‘s recommended to do in browsers), but also thought it‘d be a nice QoL improvement here to add `{numeric: true}`.

Whereas before it would sort `file1.js, file10.js, file2.js`, now it will sort `file1.js, file2.js, file10.js`.